04 DisplayData/package.json

```diff
@@ -11,24 +11,25 @@
   "dependencies": {
     "bootstrap": "^3.3.7",
     "jquery": "^3.2.1",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-router": "^3.0.5"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-router": "^4.2.0",
```

Comment:
react-router is not necessary because it's the core. I mean,  `react-router` is used in `react-router-dom`